admin/server/app/createStaticRouter.js

```diff
@@ -69,13 +69,16 @@ module.exports = function createStaticRouter (keystone) {
 	/* Prepare LESS options */
 	var elementalPath = path.join(path.dirname(require.resolve('elemental')), '..');
 	var reactSelectPath = path.join(path.dirname(require.resolve('react-select')), '..');
+	var keystoneTinymcePath = path.dirname(require.resolve('keystone-tinymce'));
 	var customStylesPath = keystone.getPath('adminui custom styles') || '';
 
 	var lessOptions = {
 		render: {
+			javascriptEnabled: true,
```

Comment:
Was [disabled (deprecated?) in `less@3`](http://lesscss.org/usage/#less-options-enable-inline-javascript-deprecated-)